src/openrct2/drawing/ttf_sdlport.c

```diff
@@ -1272,6 +1433,20 @@ static bool CharacterIsDelimiter(char c, const char *delimiters)
     return false;
 }
 
+void TTF_SetFontHinting(TTF_Font* font, int hinting)
```

Comment:
This is only called with `0` or `1` for `hinting`.

Reply:
Yes, but it is the original function backported from SDL_ttf. I figured that might be better to use, conceptually, rather than just hacking it in.